batch/create_with_labels_runnable.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START batch_labels_runnable]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	batch "cloud.google.com/go/batch/apiv1"
+	"cloud.google.com/go/batch/apiv1/batchpb"
+	durationpb "google.golang.org/protobuf/types/known/durationpb"
+)
+
+// Creates and runs a job with custom labels for runnable.
+func createJobWithCustomRunnablesLabels(w io.Writer, projectID, region, jobName string) (*batchpb.Job, error) {
+	ctx := context.Background()
+	batchClient, err := batch.NewClient(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("batchClient error: %w", err)
+	}
+	defer batchClient.Close()
+
+	// Setting some labels for runnable
+	labels := map[string]string{
+		"env":  "dev",
+		"type": "single command",
+	}
+
+	runn := &batchpb.Runnable{
+		Executable: &batchpb.Runnable_Script_{
+			Script: &batchpb.Runnable_Script{
+				Command: &batchpb.Runnable_Script_Text{
```

Comment:
nit: leave a comment above the `Command` field describing where this terminal shell script will be run.

Reply:
The command will be executed inside a container, after it's creation. Actually, there is no hidden logic, the output will be printed to the process's stdout. Since it is required to set this field, I intentionally kept it as simple as possible and focused on labeling. 
There is a helpful page in the documentation that explains how to deal with `Executable` ([link](https://cloud.google.com/batch/docs/create-run-basic-job)). Do you think it would be neat to leave link as a comment?

Reply:
Watch you have is probably fine.